From 277d5acba492b97f6de8d13d6f11bac89072e6a7 Mon Sep 17 00:00:00 2001
From: Julian Brown <julian.brown@cpanel.net>
Date: Thu, 3 Sep 2020 09:32:51 -0500
Subject: [PATCH 09/13] Verbose mkmf.

---
 lib/mkmf.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/mkmf.rb b/lib/mkmf.rb
index eabccd4..cf30689 100644
--- a/lib/mkmf.rb
+++ b/lib/mkmf.rb
@@ -1930,7 +1930,7 @@ SRC
 SHELL = /bin/sh
 
 # V=0 quiet, V=1 verbose.  other values don't work.
-V = 0
+V = 1
 Q1 = $(V:1=)
 Q = $(Q1:0=@)
 ECHO1 = $(V:1=@ #{CONFIG['NULLCMD']})
-- 
2.25.1

